source/configuration/read.txt

```diff
@@ -348,20 +359,29 @@ Change Streams
           This setting overrides the ``change.stream.lookup.full.document``
           setting.
 
-   * - ``change.stream.micro.batch.max.partition.count``
-     - | The maximum number of partitions the {+connector-short+} divides each 
-         micro-batch into. Spark workers can process these partitions in parallel.
-       |  
-       | This setting applies only when using micro-batch streams.
-       |
-       | **Default**: ``1``
-
-       .. warning:: Event Order
-
-          Specifying a value larger than ``1`` can alter the order in which
-          the {+connector-short+} processes change events. Avoid this setting
-          if out-of-order processing could create data inconsistencies downstream. 
-
+   * - ``change.stream.startup.mode``
+     - | Specifies how the connector starts up when no offset is available.
+       
+       | This setting accepts the following values:
+        
+       - ``latest``: The connector begins processing
+         change events starting with the most recent event.
+         It will not process any earlier unprocessed events.
+       - ``timestamp``: The connector begins processing change events at a specified time.
+           
+         To use the ``timestamp`` option, you must specify a time by using the
+         ``change.stream.startup.mode.timestamp.start.at.operation.time`` setting.
+         This setting accepts timestamps in the following formats:
+         
+         - An integer representing the number of seconds since the
```

Comment:
Would examples be helpful here? If so, how do you think they should be presented?

Reply:
Personally, I think it doesn't need an example because the timestamp formats are well explained and have solid linked documentation. If you do choose to add examples, I suggest using one line code blocks for each format under the description.